.github/workflows/create-documentation-issue.yml

```diff
@@ -0,0 +1,41 @@
+name: Create Documentation Issue
+on:
+  pull_request:
+    types:
+      - labeled
+env:
+  PR_NUMBER: ${{ github.event.number }}
+
+jobs:
+  create-issue:
+    if: ${{ github.event.label.name == 'needs-documentation' }}
```

Comment:
How do we avoid accidently adding this label on backports or other issues that won't make sense?

Maybe tag the user that applied the label in the new documentation issue or do you have thoughts on other mechanism to ensure good ownership after its been created?

Reply:
Let me see if we can assign the documentation issue to the contributor who created the PR. WDYT about that?